src/raster/r.viewshed.exposure/r.viewshed.exposure.py

```diff
@@ -0,0 +1,1409 @@
+#!/usr/bin/env python3
+
+"""
+MODULE:       r.viewshed.exposure
+
+AUTHOR(S):    Zofie Cimburova, Stefan Blumentrath
+
+PURPOSE:      Computes visual exposure to defined exposure source using
+              weighted parametrised cummulative viewshed analysis
+
+COPYRIGHT:    (C) 2022 by Zofie Cimburova, Stefan Blumentrath, and the GRASS
+              GIS Development Team
+
+REFERENCES:   Cimburova, Z., Blumentrath, S., 2022. Viewshed-based modelling of
+              visual exposure to urban greenery - an efficient GIS tool for
+              practical appliactions (in press). Landscape and Urban Planning.
+
+This program is free software under the GNU General Public
+License (>=v2). Read the file COPYING that comes with GRASS
+for details.
+"""
+
+# %module
+# % label: Visual exposure to defined exposure source.
+# % description: Computes visual exposure to defined exposure source using weighted parametrised cummulative viewshed analysis.
+# % keyword: raster
+# % keyword: viewshed
+# % keyword: line of sight
+# % keyword: LOS
+# % keyword: visual exposure
+# %end
+
+# %option G_OPT_R_INPUT
+# % key: dsm
```

Comment:
Consider using G_OPT_R_ELEV (key would be then elevation) or at least change key to 'input' to match r.viewshed.

Reply:
Implemented in e45b649